### PR TITLE
Adding Active Detectors Report Script

### PR DESCRIPTION
**Pull request by rcastley**

### CODE REVIEW

**Comment by rcastley** [comment]
I have read the CLA and I hereby accept the Terms

**Comment by rcastley** [comment]
I have read the Code of Conduct and I hereby accept the Terms

**Comment by greatestusername-splunk** [comment]
Thank you for the updates!

One last thing, do you mind moving this into the `integration-examples` folder along with other scripts and such? Other than that I'm good!

File: detectors/active_detectors/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+The output will show detectors with no events, detectors with events and how many events have fired within the number of days specified.
+
+Also, a CSV report will be creating in the same directory as you run the script.
+
+## Using the Active Detector Report script
+
+![Active Detectors Report](./images/screenshot.png)
+
+1. Install the required packages with `pip3 installl -r requirements.txt`
```
[the remark this answers is not in the document]
Comment (greatestusername-splunk) [reply detectors/active_detectors/README.md]: Resolved.

**Review by greatestusername-splunk — APPROVED** [review]

**Pull request merged**